book/pipeline-service.md

```diff
@@ -1,29 +1,31 @@
 # Pipeline Service
 
-Pipeline Service provides Tekton APIs and services to kcp workspaces.
-When a workspace binds to the Pipeline Service APIExport, it has access to the following Tekton APIs:
+Pipeline Service provides Tekton APIs and services to StoneSoup.
+In the initial phase of StoneSoup, Pipeline Service will be provided by a stock
+installation of the OpenShift Pipelines operator.
+This deployed version will be the a candidate build of the OpenShift Pipelines
+operator from a Red Hat build system.
 
-- `Task`
-- `Pipeline`
-- `PipelineRun`
-- `Repository` (for [Pipelines as Code](https://pipelinesascode.com))
+The deployment of the OpenShift Pipelines operator will have the following
+notable configurations:
 
-The following features are also provided:
+- Tekton Triggers will be disabled entirely.
+- The pruner (provided by the Pipelines operator) will be disabled in favor of
```

Comment:
just wondering, is max-keep-run pac feature will not used?

Reply:
Results has the ability to delete PipelineRuns/TaskRuns once they are archived. We want pipeline data to live off cluster as much as possible, and use Results to gate retention policies.